src/main/resources/files/Service.java.tpl

```diff
@@ -0,0 +1,20 @@
+package [# th:text="${java_package}"/].client;
+
+import org.eclipse.microprofile.rest.client.annotation.RegisterProvider;
```

Comment:
@rdebusscher IMHO unused import

Reply:
@rdebusscher  ?

Reply:
Missed this comment the first time. Done. Thx